File: Examples.VB/Features/Workbook/ImportExcelFileToWorkbook.vb
Namespace Features.Workbook
    Public Class ImportExcelFileToWorkbook
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            'Change the path to the real file path when open.
            workbook.Open(IO.Path.Combine(CurrentDirectory, "source.xlsx"), OpenFileFormat.Xlsx)
        End Sub
        Public Overrides ReadOnly Property CanDownload As Boolean
            Get
                Return False
            End Get
        End Property
        Public Overrides ReadOnly Property ShowViewer As Boolean
            Get
                Return False
            End Get
        End Property
        Public Overrides ReadOnly Property IsUpdate As Boolean
            Get
                Return True
            End Get
        End Property
    End Class
End Namespace

File: Examples/Features/Workbook/ImportExcelFileToWorkbook.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Workbook
{
    public class ImportExcelFileToWorkbook : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            //Change the path to the real file path when open.
            workbook.Open(System.IO.Path.Combine(this.CurrentDirectory, "source.xlsx"));

        }

        public override bool CanDownload
        {
            get
            {
                return false;
            }
        }

        public override bool ShowViewer
        {
            get
            {
                return false;
            }
        }
    }

}
